helm/examples/filestore/README.md

```diff
@@ -0,0 +1,47 @@
+# HPCC FileStore storage
+
+## Directory contents
+
+### FileStore Instance
+Filestore instances are fully managed NFS file servers on Google Cloud for use with applications running on Compute Engine virtual machines (VMs) instances or Google Kubernetes Engine clusters. You can find more information here: https://cloud.google.com/filestore/docs
+
+To create a Filestore instance: https://cloud.google.com/filestore/docs/creating-instances
+If possible create a Filestore instance in the same zone
+
+Here is a sample code
```

Comment:
Better worded as:
```
Example of creating a Filestore instance:
```

Reply:
@xwang2713 - trivial, but missed this one?

Reply:
updated